package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "raf",
-  "version": "3.4.0",
+  "version": "3.4.1",
```

Comment:
version numbers typically shouldn't be bumped in PRs

Reply:
The version number on the published package on npm is already 3.4.1 so I just bumped it to reflect that. But I can remove it?

Reply:
v3.4.1 on npm likely has more changes than just this; we should wait for the maintainer to push the v3.4.1 tag up to the repo first.

Reply:
right, but `npm publish` publishes whatever was on disk - so that means they had to bump the version locally, along with potentially other changes. only the person who published can know what those were, and can push them up and tag them properly.

Reply:
I actually diffed the two to figure out what was missing.

```
npm pack raf@3.4.1
tar -xzf raf-3.4.1.tgz
git clone https://github.com/chrisdickinson/raf.git
diff -r package/ raf/
Only in raf/: .git
Only in raf/: .gitignore
diff -r package/package.json raf/package.json
3c3
<   "version": "3.4.1",
---
>   "version": "3.4.0",
```

Reply:
alrighty, fair enough :-) it'd still probably be good to have a single taggable commit that bumps the PR version tho

Reply:
You win @ljharb 😢